src/app/models/form.server.model.ts

```diff
@@ -536,14 +536,14 @@ const compileFormModel = (db: Mongoose): IFormModel => {
   ) {
     this.msgSrvcName = msgSrvcName
 
-    return await this.save({ isNew: true, session })
+    return await this.save({ session })
```

Comment:
nit: probably do not need the `await` keyword here since we are returning the promise